tests/test_coreg.py

```diff
@@ -484,6 +485,57 @@ def test_blockwise_coreg(self, pipeline, subdivision):
         # Check that offsets were actually calculated.
         assert np.sum(np.abs(np.linalg.norm(stats[["x_off", "y_off", "z_off"]], axis=0))) > 0
 
+    def test_coreg_raster_and_ndarray_args(_) -> None:
```

Comment:
Is `_` playing a role in the test function arguments? I've only ever seen `self`

Reply:
An argument needs to be provided as it is a method, but this particular method does not actually need `self`. Potentially, this should be its own function, but currently all `Coreg`-related tests are in the `TestCoregClass`, so it felt wrong to put it outside by itself!